test/integration/cases/install-with-region/run.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source /helpers.sh
+
+VERSION=1.31
+
+mock::aws
+wait::dbus-ready
+# block access to us-west-2 ssm install url
+echo "0.0.0.0 amazon-ssm-us-west-2.s3.us-west-2.amazonaws.com" >> /etc/hosts
```

Comment:
you could verify that this works by trying to execute it without passing a region and assert it fails